### PR TITLE
feat: set commit timeout #ntrn-79

### PR DESCRIPTION
**Pull request by NeverHappened**
https://hadronlabs.atlassian.net/browse/NTRN-79

checked only by printing in server config in cmd, it's hard to discover cometbft settings in runtime

### CODE REVIEW

**Review by pr0n00gler — CHANGES_REQUESTED** [review]
Please fix linter issues

**Comment by NeverHappened** [comment]
I think it's base branch issues, waiting to rebase on fix of them

**Comment by foxpy** [comment]
@NeverHappened you have merge conflicts by the way

File: cmd/neutrond/root.go [hunk 1]
```diff
@@ -87,7 +88,14 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 			}
 
 			customTemplate, customNeutronConfig := initAppConfig()
-			return server.InterceptConfigsPreRunHandler(cmd, customTemplate, customNeutronConfig, tmcfg.DefaultConfig())
+			err = server.InterceptConfigsPreRunHandler(cmd, customTemplate, customNeutronConfig, tmcfg.DefaultConfig())
```
Comment (mmikev) [inline cmd/neutrond/root.go]: You can just create our own `DefaultConfig()` instead of `tmcfg.DefaultConfig()` with `timeout_commit = 5s`

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/5764927281

**Review by pr0n00gler — APPROVED** [review]

**Review by swelf19 — APPROVED** [review]

**Pull request merged**